examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp

```diff
@@ -57,10 +83,25 @@ void FabricSyncAddBridgeCommand::OnCommissioningComplete(chip::NodeId deviceId,
                         ChipLogValueX64(mBridgeNodeId));
 
         char command[kMaxCommandSize];
-        snprintf(command, sizeof(command), "descriptor subscribe parts-list %d %d %ld %d", kSubscribeMinInterval,
+
+        // Prepare and push the descriptor subscribe command
+        snprintf(command, sizeof(command), "descriptor subscribe parts-list %d %d %" PRIu64 " %d", kSubscribeMinInterval,
                  kSubscribeMaxInterval, mBridgeNodeId, kAggragatorEndpointId);
+        PushCommand(command);
 
+        // Prepare and push the commissioner control subscribe command
+        snprintf(command, sizeof(command),
```

Comment:
even if using snprintf, you should use `StringBuilder` instead that guarantees good sizing and null termination.